src/state/messages/events/index.tsx

```diff
@@ -36,6 +36,10 @@ export function MessagesEventBusProvider({
       // @ts-ignore
       window.bus = bus
     }
```

Comment:
Would it hurt to call `bus.resume()` here? Ideally effects should be symmetric so that running effect->cleanup->effect still works.

Reply:
Good call, added! Also removed dev debug code, no need for it really.